react/src/components/molecules/AccordionItem/index.js

```diff
@@ -88,7 +95,11 @@ AccordionItem.propTypes = {
   /** Whether to style the accordion as secondary or not. */
   secondary: PropTypes.bool,
   /** Heading level for accordion title */
-  headerLevel: PropTypes.number
+  headerLevel: PropTypes.number,
+  /** Expand button id, used for a11y */
+  buttonId: PropTypes.string,
```

Comment:
can we only ask for one ID prop and inside the component form the two different ids? 
e.g.  something like:
```
prop: accordionID
expandableSectionId = `expandable${accordionID}`
labelId = `label${accordionID}`
```